cdap-docs/admin-manual/source/security/impersonation.rst

```diff
@@ -8,6 +8,39 @@
 Configuring Impersonation
```

Comment:
let's rename this to just ``Impersonation``, since there's more in this document than configuration?